### PR TITLE
[INT-146] Add applicant hook request model

### PR DESCRIPTION
**Pull request by KlochkovHF**
https://huntflow.atlassian.net/browse/INT-146
Added applicant webhook request model

### CODE REVIEW

File: models/base_hook.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+from pydantic import BaseModel, Field
+
+from models.consts import WebhookEventType
+
+
+class Account(BaseModel):
+    id: int = Field(..., description="Account ID", example=1)
+    name: str = Field(..., description="Account owner name", example="John")
+    nick: str = Field(..., description="Account nick", example="Test")
```
Comment (aleks-v-k) [inline models/base_hook.py:11]: It's not an user's account in `meta`. So, it makes sense to add docstring to the model, and `example="John"` is wrong

File: models/base_hook.py
Comment (aleks-v-k) [inline models/base_hook.py:1]: base_hook.py -> base.py

File: models/common_models/applicant_base.py
Comment (aleks-v-k) [inline models/common_models/applicant_base.py:1]: postfix `_base` looks redundant here (in common_models). All modules are `_base`

File: models/common_models/applicant_base.py [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+class Applicant(BaseModel):
+    id: int = Field(..., description="Applicant ID", example=1)
+    email: Optional[str] = Field(None, description="Applicant's email", example="email@example.com")
+    first_name: str = Field(..., description="Applicant's firstname", example="test_name")
+    last_name: str = Field(..., description="Applicant's lastname", example="test")
+    middle_name: Optional[str] = Field(None, description="Applicant's patronymic", example="test")
+    position: Optional[str] = Field(None, description="Applicant's current job", example="test")
+    birthday: Optional[date] = Field(
+        None,
+        description="The last position in which the work of the applicant",
```
Comment (aleks-v-k) [inline models/common_models/applicant_base.py:103]: wrong description

File: models/common_models/applicant_base.py [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+        example=date(1970, 1, 1),
+    )
+    company: Optional[str] = Field(
+        None,
+        description="Last company the applicant worked for",
+        example="test",
+    )
+    money: Optional[str] = Field(None, description="Desired salary of the applicant", example="10$")
+    phone: Optional[str] = Field(None, description="Applicant's phone", example="+99999999")
+    skype: Optional[str] = Field(None, description="Applicant,s skype", example="test_skype")
```
Comment (polina-koval) [inline models/common_models/applicant_base.py:113]: Applicant's instead of Applicant,s.

File: models/common_models/vacancy_base.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    )
+    account_region: Optional[int] = Field(None, description="Vacancy region ID", example=1)
+    applicant_to_hire: Optional[int] = Field(
+        None,
+        description="Amount applicants to hire",
+        example=1,
+    )
+    body: Optional[str] = Field(
+        None,
+        description="Vacancy responsivbilities (HTML)",
```
Comment (polina-koval) [inline models/common_models/vacancy_base.py:49]: Typo? responsibilities?

**Review by aleks-v-k — APPROVED** [review]

**Pull request merged**